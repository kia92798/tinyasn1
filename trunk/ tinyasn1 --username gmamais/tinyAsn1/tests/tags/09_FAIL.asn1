MY-MODULE DEFINITIONS ::= BEGIN

-- tag glass because a is optional

	MySQ0 ::= SEQUENCE {
		a	A	OPTIONAL, 
		b	B
	}

	A::= CHOICE {
		d	[0]	NULL,
		e	[1]	NULL
	}
	
	B::= CHOICE {
		d	[0]	NULL,
		e	[1]	NULL
	}

END
