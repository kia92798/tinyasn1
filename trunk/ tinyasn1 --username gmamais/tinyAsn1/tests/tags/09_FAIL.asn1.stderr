Error: Tag clash for type defined in line 5.The type contains has two children (or grandchilden a choice child) with the same tag. The child that clashes are located in lines: 11 and 16
